utils/tests/test_enhanced_logger.py

```
@@ -0,0 +1,1193 @@
+
+            assert "context" in data
+            assert data["context"]["correlation_id"] == "test-123"
+            assert data["context"]["user_id"] == "user-456"
+
+    def test_format_with_exception(self):
+        """Test formatting with exception information."""
+        config = FormatterConfig()
+        formatter = EnhancedJsonFormatter(config)
+
+        try:
+            raise ValueError("Test exception")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)